Example/NativeBase-KitchenSink/src/screens/toast/toast-position.js

```diff
@@ -42,7 +43,10 @@ class ToastPosition extends Component {
               Toast.show({
                 text: "Wrong password!",
                 buttonText: "Okay",
-                position: "top"
+                position: "top",
+                onClose: () => {
+                  TLTRN.logScreenLayout("Wrong password!");
```

Comment:
Is there a way to capture automatically? So customer does not need to add one line in onClose?